### PR TITLE
remove parameters

### PR DESCRIPTION
**Pull request by pushkala-datastax**

### CODE REVIEW

File: driver-http/src/main/resources/activities/baselines/http-iot.yaml [hunk 1]
```diff
@@ -27,8 +27,6 @@ bindings:
 blocks:
   - tags:
       phase: schema
-    params:
-      prepared: false
```
Comment (jshook) [inline driver-http/src/main/resources/activities/baselines/http-iot.yaml:31]: This params property sets the default for any statements in this block, which is only DDL. If the statements are prepared (true by defult), then they will be rejected, as prepared statements need a schema to be checked against.

Comment (jshook) [reply driver-http/src/main/resources/activities/baselines/http-iot.yaml:31]: Maybe we can find a better way to handle this in terms of defaults and user feedback.

Comment (jshook) [reply driver-http/src/main/resources/activities/baselines/http-iot.yaml:31]: Can we close this PR and discuss the issue in more detail?

**Comment by pushkala-datastax** [comment]
Yes please.

File: driver-http/src/main/resources/activities/baselines/http-iot.yaml (hunk 1 above)
Comment (pushkala-datastax) [reply driver-http/src/main/resources/activities/baselines/http-iot.yaml:31]: yes plz

**Pull request closed without merging**